packages/teleport/src/services/audit/types.ts

```diff
@@ -908,6 +1016,15 @@ type RawEventAuthFailure<T extends EventCode> = RawEvent<
   }
 >;
 
+type RawEventSFTP<T extends EventCode> = RawEvent<
+  T,
+  {
+    path: string;
+    error: string;
+    ['addr.local']: string;
```

Comment:
Does `['server_hostname']: string;` need to be added as well?

Reply:
oh you're right, i was wondering how it bypassed the type check but it got bypassed since we used bracket string notation. that's frustrating b/c what is the point of the type-check then :laughing: 

i'll put a fix out, but it's not urgent and doesn't affect production code